x/ibc/spec/01_concepts.md

```diff
@@ -45,3 +45,33 @@ identifier, but differing feature sets. This will result in undefined behavior
 with regards to version selection in `ConnOpenTry`. Each version in a set of 
 versions should have a unique version identifier.
 :::
+
+### Channel Version Negotation
+
+During the channel handshake procedure a version must be agreed upon between
+the two parties. The selection process is largely left to the callers and 
+the verification of valid versioning must be handled by application developers
+in the channel handshake callbacks.
+
+During `ChanOpenInit`, a version string is passed in and set in party A's 
+channel state.
+
+During `ChanOpenTry`, a version string and counterparty version string are 
```

Comment:
Maybe worth noting that which chain is the counterparty switches between this and the next paragraph, it's a bit confusing to read, might be helpful to label them A/B